roles/calico_master/tasks/main.yml

```diff
@@ -23,7 +23,7 @@
     -f {{ mktemp.stdout }}/calico-policy-controller.yml
     --config={{ openshift.common.config_base }}/master/admin.kubeconfig
   register: calico_create_output
-  failed_when: ('already exists' not in calico_create_output.stderr) and ('created' not in calico_create_output.stdout)
+  failed_when: "('already exists' not in calico_create_output.stderr) and ('created' not in calico_create_output.stdout) and calico_create_output.rc != 0"
```

Comment:
Do you mind changing all of these conditionals to be lists? We're trying to convert all conditionals to lists whenever we touch them.

```
failed_when:
  - a 
  - not b
  - c or d
```

Reply:
I don't think its possible in this case. The list would turn these `and` in `or`'s effectively.

If its just for readability I could use slashes probably

Reply:
No, lets not do slashes. The way I'm reading that conditional it'd just be
```
failed_when:
  - 'already exists' not in calico_create_output.stderr
  - 'created' not in calico_create_output.stdout
  - calico_create_output.rc != 0
```

Anyway, we'll minimize change and accept it as is.